src/xdocs/index.xml.vm

```diff
@@ -119,6 +119,50 @@
       </p>
     </section>
 
+    <section name="JRE and JDK">
+      <p>
+        Runtime of Checkstyle is limited only by minimal version or JRE.
+      </p>
+      <div class="wrapper">
+        <table>
+          <tr>
+            <th>Checkstyle version</th>
+            <th>JRE version</th>
+          </tr>
+          <tr>
+            <td>
+              7.x, 8.x, 9.x
```

Comment:
Does it make sense to also say here `7.x and above` like you did for the JRE version? At the least maybe say `7.x through current release`.

Reply:
I thought on this but I found this more descriptive and clear.

If somebody else think is good I will change.

Reply:
>beginning_development.html

No changes as it already has note on required jdk.
Do you want explicit reference to jdk 17 ? As I did  in index page ?

Reply:
> No changes as it already has note on required jdk.

https://checkstyle.org/beginning_development.html#Before_development
> Ensure that Git, Java JDK >= 1.8 are installed.

I am looking for the max version we support. We weren't supporting JDK ¿17? until we removed powermock.

Unless we are future proof for more versions to come, I suggest:
> Ensure that Git, Java JDK >= 1.8 are installed. 17 is the highest JDK we support since last updated. Please report an issue if there is a problems with newer JDKs.

Reply:
I added jdk 17 note at https://github.com/checkstyle/checkstyle/pull/11093 to avoid conflicts between such PRs
jdk17 is already supported after merge of 682a3667ccbff05df3ce27c4f88514a45590fa36